.env.example

```diff
@@ -34,6 +34,10 @@ VUE_APP_MAPBOX_TOKEN=
 VUE_APP_NOMINATIM_HOST=
 VUE_APP_NOMINATIM_KEY=
 
+# Automatically retrigger the search based on map center
+# Explicitly disable it by setting to false
+VUE_APP_SEARCH_AFTER_MAP_MOVE=
```

Comment:
Name does not convey it is an *opt-out*.
Something like `VUE_APP_DISABLE_AUTO_SEARCH_ON_MAP_MOVE` may be better.

Comment could also be more precise:

```
# A checkbox in UI lets users enable automatic search as they move the map.
# You can disable this feature by setting this variable to 'true'
```

Reply:
Yes, thanks.